src/group/group.service.ts

```diff
@@ -28,6 +32,41 @@ export class GroupService {
     return group;
   }
 
+  async findGroupWithBookmarks(
+    user: User,
+    groupId: string,
+    page: number,
+  ): Promise<FindOneRes> {
+    const group = await this.findOneWithUser(groupId);
+
+    if (!this.isOwner(user, group)) {
+      throw new UnauthorizedException();
+    }
+
+    // FIXME: Fix magic number and Set count per a page.
+    const take = 9;
+    const skip = (page - 1) * take;
+
+    const [bookmarks, count] = await this.bookmarkRepository.findAndCount({
+      where: { group: { id: groupId } },
+      skip,
+      take,
+      // FIXME: Get order query
```

Comment:
I saw so many FIXME comments for three requested reviews.
Have any specific plans to resolve it?

Reply:
1. `take` in pagination

Once we decide how many bookmarks to get, I will manage this in a config file.

2. `order`, `search`, `tag`

I will do next. I think it's too many jobs to do in a PR.